<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemManagement</class>
 <widget class="QDialog" name="SystemManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>4</y>
     <width>380</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>시스템 관리</string>
   </property>
   <widget class="QPushButton" name="btnSystemStop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>정지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSystemRun">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReboot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>시스템 재부팅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpgrade">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>업그레이드</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>380</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>자료 관리</string>
   </property>
   <widget class="QPushButton" name="btnDataRestore">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>USB로부터 복원</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDataSave">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>USB에 저장</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>380</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>설정데이터 관리</string>
   </property>
   <widget class="QPushButton" name="btnSetupSave">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>USB에 저장</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetupRestore">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>USB로부터 복원</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>537</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>380</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>초기화</string>
   </property>
   <widget class="QPushButton" name="btnFactoryReset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>전체초기화</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInitData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>데이터초기화</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInitTmsData">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>T 데이터초기화</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>591</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteInterrupt">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>541</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I 데이터 삭제</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>456</y>
     <width>380</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>원격지원</string>
   </property>
   <widget class="QPushButton" name="btnRemoteStop">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>56</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>원격지원 정지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoteRun">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>56</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>원격지원 실행</string>
    </property>
   </widget>
   <widget class="IpEdit" name="ReverseProxyIP">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>211.238.253.248</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>21</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="remoteStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>116</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PORT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="NumEdit" name="ReverseProxyPort">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IpEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">ipedit.h</header>
  </customwidget>
  <customwidget>
   <class>NumEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">numedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>SystemManagement</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
